### PR TITLE
add angler to wiki

### PR DESCRIPTION
**Pull request by PixelBoot**

### CODE REVIEW

File: _data/devices/angler.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+cpu_freq: 2.5 GHz
+custom_unlock_cmd: fastboot oem let-me-free
+depth: 7.3 mm (0.29 in)
+download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Home</kbd>
+  + <kbd>Power</kbd>.
+gpu: Adreno 330
+height: 144.6 mm (5.69 in)
+install_method: heimdall
+maintainers:
+- People1
```
Comment (Hasaber8) [inline _data/devices/angler.yml:32]: hahahah

File: _data/devices/angler.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- People2
+models:
+- A0001
+- SM-G900F
```
Comment (Hasaber8) [inline _data/devices/angler.yml:36]: oh no

File: _data/devices/angler.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+name: Galaxy Note Sample
```
Comment (Hasaber8) [inline _data/devices/angler.yml:37]: yes

**Comment by Hlcpereira** [comment]
https://wiki-preview.pixelexperience.org/42/4e392add/devices/angler/

File: _data/devices/angler.yml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+network:
+- 2G GSM
+- 2G CDMA
+- 3G UMTS
+- 3G CDMA2000
+- 4G LTE
+peripherals:
```
Comment (Hasaber8) [inline _data/devices/angler.yml:44]: recheck

**Review by Hasaber8 — CHANGES_REQUESTED** [review]

**Pull request merged**